src/main/java/org/jenkinsci/test/acceptance/po/DumbSlave.java

```diff
@@ -30,12 +32,28 @@ public DumbSlave(Jenkins j, String name) {
      * Selects the specified launcher, and return the page object to bind to it.
      */
     public <T extends ComputerLauncher> T setLauncher(Class<T> type) {
+        try {
+            selectType(type);
+        } catch (NoSuchElementException x) {
+            // Possibly need to install split command-launcher.
+            save();
+            try {
+                getJenkins().getPluginManager().installPlugins(new PluginSpec("command-launcher", null));
```

Comment:
Maybe it makes sense to check the Launcher's class even before `selectType()`. Otherwise you try to install the plugin even for other Launcher types